common/functions/src/scalars/conditionals/in_basic.rs

```diff
@@ -172,6 +187,12 @@ impl<const NEGATED: bool> Function for InFunction<NEGATED> {
             TypeID::Float64 => {
                 float_contains!(f64, input_col, input_rows, columns, least_super_dt);
             }
+            TypeID::Date => {
+                scalar_contains!(i32, input_col, input_rows, columns, least_super_dt)
+            }
+            TypeID::DateTime => {
+                scalar_contains!(i64, input_col, input_rows, columns, least_super_dt);
+            }
             _ => {
                 unimplemented!()
```

Comment:
Better to return error instead of the unimplemented panic.

Reply:
yes, i will open a new pr to fix it.

Reply:
Great, seems some `unimplemented` both in the `common/functions` and `common/datavalues`.

Reply:
Yes, because there are some judges before and the program can't reach `unimplementd`. But I think return error seems better.